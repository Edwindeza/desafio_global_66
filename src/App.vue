<template>
  <div id="app">
    <router-view/>
    <Loader></Loader>
  </div>

</template>

<script>

import Loader from '@/components/Loader/Loader.vue';

export default ({
  name: 'App',
  components: {
    Loader,
  },
});
</script>
<style lang="scss">
body{
  margin: 0;
}
</style>
