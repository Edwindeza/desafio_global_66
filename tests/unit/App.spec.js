import { shallowMount } from '@vue/test-utils';
import App from '@/App.vue';

describe('Prueba App Component', () => {
  test('Debe de hacer match con el snapshot', () => {
    const wrapper = shallowMount(App);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
