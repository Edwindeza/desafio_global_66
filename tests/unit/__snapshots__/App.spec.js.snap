// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba App Component Debe de hacer match con el snapshot 1`] = `
<div id="app">
  <router-view></router-view>
  <loader-stub></loader-stub>
</div>
`;
